charts/fluentd/values.yaml

```diff
@@ -53,6 +53,14 @@ resources: {}
   # limits:
   #   memory: 128Mi
 
+## only available if kind is Deployment
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
```

Comment:
Can you add a generic field where we can write a full HPA rule?
maybe under extraRules: and leave an example commented.